ci-operator/step-registry/telco5g/ptp-cluster-setup/telco5g-ptp-cluster-setup-commands.sh

```diff
@@ -51,7 +48,27 @@ cat << EOF > ~/ocp-install.yml
   hosts: hypervisor
   gather_facts: false
   tasks:
-
+EOF
+if [[ "$JOB_TYPE" == "periodic" ]]; then
+cat << EOF >> ~/ocp-install.yml
+  - name: Check if abort file exists
+    stat:
+      path: /home/kni/abort
+    register: file_info
+    failed_when: file_info.stat.exists
+
+  - name: Handle file existence
+    debug:
+      msg: "aborting run"
+    when: file_info.stat.exists
+
+  - name: Handle file existence
+    debug:
+      msg: "continue run"
+    when: file_info.stat.exists == False
+EOF
+fi
+cat << EOF >> ~/ocp-install.yml
   - name: Wait 300 seconds, but only start checking after 10 seconds
     wait_for_connection:
```

Comment:
This is the task for connection to hypervisor, it should be the first one. Please put your tasks after it.

Reply:
fixed